docs/docs/flags/output.1.md

```diff
@@ -48,7 +48,10 @@ Other options:
   - **exec-env**: When tracing execve/execveat, show the environment variables that were used for execution.
   - **relative-time**: Use relative timestamp instead of wall timestamp for events.
   - **exec-hash**: When tracing some file related events, show the file hash (sha256).
-    - Affected events: sched_process_exec, shared_object_loaded
+    - Affected events: *sched_process_exec*, *shared_object_loaded*
+    - **pathname** option recalculates the file hash if the inode's creation time (ctime) differs, which can occur in different namespaces even for identical pathnames.
```

Comment:
Hmm I don't think this documentation is correct.
This option means that the key for finding the hash is the path of the file alone.
Either way, the ctime is checked for all options and if it doesn't match the hash is recalculated.
To be honest, I am not sure what the `pathname` option is good for. You want it for same container deployment, where the path for the same file is constant?